src/js/App.vue

```diff
@@ -23,7 +23,5 @@ export default {
 </script>
 
 <style lang="scss">
-#app {
-  @import "../sass/app";
-}
```

Comment:
@dvdherron I think the intention around nesting this inside `#app` is to be sure any styles from this component don't pollute the global styles of the application where this is eventually used. Is there another way we should do that, since this style block is not `scoped`?

Reply:
aha, that makes sense. I'll move things back to the way you had it set up at first and see if there's a better way.

Reply:
Yeah, I'm not sure. The component I copied this pattern from was written a number of years ago, so maybe @mirisuzanne has a better solution?